### PR TITLE
A Container Is a Linux Namespace and Networking Basics Post

### PR DESCRIPTION
**Pull request by everythingeverywhere**
images weren't displaying and had to abandon github recommended and just add the raw URL to image

### CODE REVIEW

File: content/blog/container-net-basics.md [hunk 1]
```diff
@@ -0,0 +1,416 @@
+
+---
+title: "A Container Is a Linux Namespace and Networking Basics"
+featured: true
+description: >
+    Demystify containers and cover some networking basics
+date: 2020-26-21
```
Comment (tybritten) [inline content/blog/container-net-basics.md:7]: Fix the date to today please

File: content/blog/container-net-basics.md [hunk 2]
```diff
@@ -0,0 +1,416 @@
...
+- Deployment
+# Author(s)
+team: 
+- Ivan Tarin
+---
+
+# A Container Is a Linux Namespace and Networking Basics
+
+![open source image @freddygthatsme see sources
+](https://raw.githubusercontent.com/everythingeverywhere/tanzu-dev-portal/main/static/images/blogs/container-net-basics/freddy-g-1i1zPbkQJNg-unsplash.jpg)
```
Comment (tybritten) [inline content/blog/container-net-basics.md:18]: the image links should be like `/images/blogs/container-net-basics/freddy-g-1i1zPbkQJNg-unsplash.jpg`

**Comment by everythingeverywhere** [comment]
git c48e255 has the latest changes which should be part of the PR

File: content/blog/container-net-basics.md [hunk 3]
```diff
@@ -0,0 +1,414 @@
+
+---
+title: "A Container Is a Linux Namespace and Networking Basics"
+featured: true
+description: >
+    Demystify containers and cover some networking basics
+date: 2020-29-6
```
Comment (tybritten) [inline content/blog/container-net-basics.md:7]:
```suggestion
date: 2020-06-29
```

File: content/blog/container-net-basics.md [hunk 4]
```diff
@@ -0,0 +1,414 @@
...
+- Deployment
+# Author(s)
+team: 
+- Ivan Tarin
+---
+
+# A Container Is a Linux Namespace and Networking Basics
+
+![open source image @freddygthatsme see sources
+](../../static/images/blogs/container-net-basics/freddy-g-1i1zPbkQJNg-unsplash.jpg)
```
Comment (tybritten) [inline content/blog/container-net-basics.md:18]:
```suggestion
](/images/blogs/container-net-basics/freddy-g-1i1zPbkQJNg-unsplash.jpg)
```

File: content/blog/container-net-basics.md [hunk 5]
```diff
@@ -0,0 +1,414 @@
...
+```
+
+ip link list
+
+```
+
+This should print out the devices that are available; any networking device that has a driver loaded can be classified as an available device. In the output, you mihght see devices like `lo` and `enp0s2`. The `ip link` command will also output two lines for each device, the link status, and the characteristics.
+
+![screenshot of output
+](../../static/images/blogs/container-net-basics/screenshot-ip-output.png)
```
Comment (tybritten) [inline content/blog/container-net-basics.md:95]:
```suggestion
](/images/blogs/container-net-basics/screenshot-ip-output.png)
```

File: content/blog/container-net-basics.md [hunk 6]
```diff
@@ -0,0 +1,414 @@
...
+### Create a Virtual Ethernet Cable for Each Namespace
+
+A veth device is a virtual ethernet device that you can think of as a real ethernet cable connecting two other devices. Virtual ethernet devices act as tunnels between network namespaces. They create a bridge to a physical network device in another namespace. Virtual ethernets can also be used as standalone network devices as well. 
+
+Veth devices are always created in interconnected pairs where packets transmitted on one device in the pair are immediately received on another device. When either device is down, the link state of the pair is down as well.
+
+In our example, we are creating two veth pairs. The jelly namespace will connect to the veth-jelly end of the cable, and the other cable end should attach to a bridge that will create the network for our namespaces. We create the same cable and connect it to the bridge on the pb side.
+
+
+![pb&jelly diagram](../../static/images/blogs/container-net-basics/containers.png)
```
Comment (tybritten) [inline content/blog/container-net-basics.md:125]:
```suggestion
![pb&jelly diagram](/images/blogs/container-net-basics/containers.png)
```

**Pull request merged**